root/context.js

```diff
@@ -7,8 +7,10 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-import React from 'react';
-import type {ComponentType} from 'react';
+// NOTE: Don't convert to an ES module; this is used by root/server.js.
```

Comment:
Additionally, ESLint rules should be changed somehow (via either local comment or configuration file) to https://eslint.org/docs/rules/#nodejs-and-commonjs

Reply:
I don't think we use any of those ATM, but I added some `eslint-disable import/no-commonjs` comments to disable the current errors.